service/core/kafka/client/src/main/scala/com/lightbend/lagom/internal/broker/kafka/KafkaSubscriberActor.scala

```diff
@@ -122,11 +126,24 @@ private[lagom] class KafkaSubscriberActor[Message](
 
 object KafkaSubscriberActor {
   def props[Message](
-    kafkaConfig: KafkaConfig, consumerConfig: ConsumerConfig, locateService: String => Future[Option[URI]],
-    topicId: String, flow: Flow[Message, Done, _],
-    consumerSettings: ConsumerSettings[String, Message], subscription: AutoSubscription,
-    streamCompleted: Promise[Done]
+    kafkaConfig:    KafkaConfig,
+    consumerConfig: ConsumerConfig,
+    locateService:  String => Future[Seq[URI]],
+    topicId:        String, flow: Flow[Message, Done, _],
```

Comment:
I think you can configure the IntelliJ auto-formatter to do this, too (using the "chop down if long" setting for line wrapping).